lib/index.js

```diff
@@ -412,8 +409,12 @@ export function createUnifiedLanguageServer({
       return
     }
 
-    const [file] = await processDocuments([document], true)
-    const result = String(file)
+    const files = await processDocuments([document], true)
+    if (files.length !== 1) {
```

Comment:
I'm not sure I follow this.
Is no document bad, more than one document bad, or both?

Reply:
The number of output files is usually equivalent to the number of input files, which is always 1 here. However, files are ignored if they are not part of a workspace, meaning sometimes the length is 1.

I rewrote it using `if (file)` instead. Maybe that’s a bit clearer, it’s closer to the original code anyway.